src/test/java/org/apache/commons/collections4/bloomfilter/WrappedBloomFilterTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.collections4.bloomfilter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.Test;
+
+public class WrappedBloomFilterTest extends AbstractBloomFilterTest<WrappedBloomFilter> {
+
+    @Override
+    protected WrappedBloomFilter createEmptyFilter(Shape shape) {
+        return new WrappedBloomFilter(new DefaultBloomFilterTest.SparseDefaultBloomFilter(shape)) {
+        };
+    }
+
+    @Test
+    public void testCharacteristics() {
```

Comment:
I do not know why but the characteristics method had no coverage when I ran tests locally. I changed this to:
```java
@ParameterizedTest
@ValueSource(ints = {0, 1, 34})
public void testCharacteristics(int characteristics) {
    Shape shape = getTestShape();
    BloomFilter inner = new DefaultBloomFilterTest.SparseDefaultBloomFilter(shape) {
        @Override
        public int characteristics() {
            return characteristics;
        }
    };
    WrappedBloomFilter underTest = new WrappedBloomFilter(inner) {};
    assertEquals(characteristics, underTest.characteristics());
}
```
Now I see coverage.